server/templates/base_page.html

```diff
@@ -1,6 +1,13 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+    <script async>
```

Comment:
I don't think this works.

When running in production, we don't load all the dependencies of each file at runtime via AJAX like we do in dev. We use the r.js optimizer to concatenate files, so this needs to be known at *build* time for production.

As an aside, `async` shouldn't do anything on an inline script tag. https://developer.mozilla.org/en/docs/Web/HTML/Element/script

> async HTML5
> Set this Boolean attribute to indicate that the browser should, if possible, execute the script > asynchronously. It has no effect on inline scripts (i.e., scripts that don't have the src
> attribute).